### PR TITLE
detect-flowbits/v3: add tests

### PR DESCRIPTION
**Pull request by TheKharleeci**
Adds test for FlowBitsTestSig6-11

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

Previous PR: #696

### CODE REVIEW

**Comment by TheKharleeci** [comment]
Corresponding Suricata PR: OISF/suricata#6899

**Comment by TheKharleeci** [comment]
I still need to add filters for the sigs that don't alert. I'm not sure why those builds are failing.

**Comment by inashivb** [comment]
> I still need to add filters for the sigs that don't alert. I'm not sure why those builds are failing.

I see
```
[1038387] 31/1/2022 -- 07:27:13 - (detect-flowbits.c:189) <Error> (DetectFlowbitParse) -- [ERRCODE: SC_ERR_INVALID_SIGNATURE(39)] - spaces not allowed in flowbit names
[1038387] 31/1/2022 -- 07:27:13 - (detect-engine-loader.c:184) <Error> (DetectLoadSigFile) -- [ERRCODE: SC_ERR_INVALID_SIGNATURE(39)] - error parsing signature "alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:7;)" from file /home/ard/Orgs/oisf/suricata-verify/tests/detect-flowbits/test.rules at line 6
```

Flowbit ORing was not supported in `5.0.x`.. as a result of that, engine seems to think `fb1 | fb2` as one flowbit and fails because spaces are not allowed.

File: tests/detect-flowbits/test.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+args:
+- -k none
```
Comment (inashivb) [inline tests/detect-flowbits/test.yaml:2]: min-version: 6 would be required here..

**Review by inashivb — CHANGES_REQUESTED** [review]

File: tests/detect-flowbits/test.rules [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb3|fb4; sid:3;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb5; sid:5;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb5|fb6; sid:6;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:7;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow; sid:8;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow2; sid:9;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:unset,myflow2; sid:10;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:noalert; flowbits:toggle,myflow2; content:"GET "; sid:11;)
+alert ip any any -> any any (msg:"Noalert"; flowbits:noalert; content:"GET "; sid:12;)
+alert ip any any -> any any (msg:"isset option"; flowbits:isset,fbt; content:"GET "; sid:13;)
```
Comment (inashivb) [inline tests/detect-flowbits/test.rules:12]: I do not see any tests for `AND` op among flowbits. Please add those rules too. e.g. `isset,fb1,fb2,..`
You can read more here: https://unixia.wordpress.com/2020/05/17/more-checks-with-lesser-rules-a-flowbit-magic-tale/

**Review by inashivb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**